.docker/scholarly-articles/data/article1/1.xml

```diff
@@ -512,7 +515,7 @@
                             foreshortened; the table upon which the bones are arranged is 120-cm wide for scale.
                         </jats:p>
                     </jats:caption>
-                    <jats:graphic xlink:href="https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif/full/1234,/0/default.jpg"/>
+                    <jats:graphic xlink:href="elife-09560-fig1-v1-1484w.jpg"/>
```

Comment:
resized formats are not used anymore, but the files were left there; refer to `elife-09560-fig1-v1.jpg` which is the full size anyway but it's currently still being produced for all articles.

Reply:
I thought Lens was still using them, but just checked and it's not (😳).

Anyway, this is only temporary (both size and format).